provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerTransformer.java

```diff
@@ -49,25 +51,34 @@ Address address(ProviderResponse.Value value) {
 
   @Override
   public Practitioner apply(PractitionerWrapper ppmsData) {
-    ProviderResponse.Value response = ppmsData.providerResponse().value().get(0);
-    ProviderContactsResponse.Value contacts;
-    if (ppmsData.providerContactsResponse().value().size() > 0) {
-      contacts = ppmsData.providerContactsResponse().value().get(0);
-    } else {
-      contacts = null;
-    }
+    ProviderResponse.Value providerResponse = ppmsData.providerResponse().value().get(0);
+    ProviderContactsResponse.Value providerContacts =
+        ppmsData.providerContactsResponse().value() == null
+                || ppmsData.providerContactsResponse().value().isEmpty()
+            ? null
+            : ppmsData.providerContactsResponse().value().get(0);
```

Comment:
This may be beyond the scope of this story, but... there are a lot of cases where a wrapper class contains an instance of a PPMS response object, but only uses the first `value()` inside it. In those cases, consider changing the wrapper to contain just the single instance of that value, instead of the whole response object. In this example, `PractitionerWrapper` has an instance of `ProviderContactsResponse`, but could really have just `ProviderContactsResponse.Value` instead. That would simplify this code, and it may simplify the controller code as well, for cases where an artificial response object gets created around a single `value`.

Reply:
I can make this change, but it won't matter once we change to NPI as a data source(I think this will be soon). If you wan't me to go ahead and make that change, just let me know